test/libsolidity/semanticTests/modifiers/stacked_return_with_modifiers.sol

```diff
@@ -1,22 +0,0 @@
-contract C {
```

Comment:
This was a 100% copy of break_in_modifier.sol. What was it supposed to do?

Reply:
Looks like it's been like that ever since this was added to end to end tests 4 years ago: 9c83109549547c8fe308969f924d3c07ce2cac76

Judging by the name, I think that this was supposed to test a function having multiple modifiers and each of them doing a return. Maybe it would be better to add such a test instead of removing this?

Reply:
Can you type up what you mean? I'll add it in.

Reply:
Sure.

Something along these lines:

```solidity
contract C {
    uint public counter;

    modifier m() {
        for (uint256 i = 0; i < 10; i++) {
            _;
            ++counter;
            return;
        }
    }

    function f() public m m m returns (uint) {
        for (uint256 i = 0; i < 10; i++) {
            ++counter;
            return 42;
        }
    }
}
// ----
// counter() -> 0
// f() -> 42
// counter() -> 4
```